zcash_client_backend/src/keys.rs

```diff
@@ -139,6 +143,8 @@ pub struct UnifiedFullViewingKey {
     #[cfg(feature = "transparent-inputs")]
     transparent: Option<legacy::AccountPubKey>,
     sapling: Option<sapling_keys::DiversifiableFullViewingKey>,
+    orchard: Option<orchard::keys::FullViewingKey>,
+    unknown: Vec<(u32, Vec<u8>)>,
```

Comment:
It's a public struct in a library crate, so I don't understand why clippy thinks this field can't be read. Consider suppressing the warning.

Reply:
Nope, this is actually a bug! We are parsing `unknown` from the encoding, but not including it when serializing. I will fix.